backend/data_wizard/views.py

```
@@ -217,6 +226,134 @@ def _process_applied_controls(self, request, records, folders_map, folder_id):
+                        if record.get("severity") != "":
+                            severity = SEVERITY_MAP.get(record.get("severity"), -1)
+                        finding_data.update({"severity": severity})
+
+                        finding_writer = FindingWriteSerializer(
+                            data=finding_data, context={"request": request}
+                        )
+                        if finding_writer.is_valid(raise_exception=True):
+                            finding_writer.save()
+                            results["successful"] += 1
+                        else:
+                            logger.info(f"Data validation failed for finding creation")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)